<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TopicView</class>
 <widget class="QWidget" name="TopicView">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>922</width>
    <height>552</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QScrollArea" name="scrollArea">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>220</y>
     <width>191</width>
     <height>281</height>
    </rect>
   </property>
   <property name="widgetResizable">
    <bool>true</bool>
   </property>
   <widget class="QWidget" name="scrollAreaWidgetContents">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>189</width>
      <height>279</height>
     </rect>
    </property>
    <widget class="QListWidget" name="topicListWidget">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>0</y>
       <width>181</width>
       <height>281</height>
      </rect>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QPushButton" name="addNewTopic">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>80</y>
     <width>151</width>
     <height>71</height>
    </rect>
   </property>
   <property name="text">
    <string>Add New Topic</string>
   </property>
  </widget>
  <widget class="QPushButton" name="addNewTopic_2">
   <property name="geometry">
    <rect>
     <x>530</x>
     <y>80</y>
     <width>151</width>
     <height>71</height>
    </rect>
   </property>
   <property name="text">
    <string>Delete Topic</string>
   </property>
  </widget>
  <widget class="QPlainTextEdit" name="searchTopicText">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>90</y>
     <width>191</width>
     <height>51</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>50</y>
     <width>181</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Search Topic:</string>
   </property>
  </widget>
  <widget class="QScrollArea" name="scrollArea_2">
   <property name="geometry">
    <rect>
     <x>490</x>
     <y>220</y>
     <width>191</width>
     <height>281</height>
    </rect>
   </property>
   <property name="widgetResizable">
    <bool>true</bool>
   </property>
   <widget class="QWidget" name="scrollAreaWidgetContents_2">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>189</width>
      <height>279</height>
     </rect>
    </property>
    <widget class="QListWidget" name="topicListSelectedWidget">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>0</y>
       <width>181</width>
       <height>281</height>
      </rect>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>300</y>
     <width>151</width>
     <height>101</height>
    </rect>
   </property>
   <property name="text">
    <string>Selected Topics &gt;&gt;</string>
   </property>
  </widget>
  <widget class="QPushButton" name="addTopicsButton">
   <property name="geometry">
    <rect>
     <x>720</x>
     <y>320</y>
     <width>151</width>
     <height>71</height>
    </rect>
   </property>
   <property name="text">
    <string>Add Selected Topics</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
